jni/src/init.c

```diff
@@ -21,6 +21,18 @@ JNIEnv* get_jni_env() {
     return env;
 }
 
+void release_jni_env(){
+    // detach current thread from the JVM
+    //   FIXME: this should be replaces with some sort of
+    //          lazy detachment. 
+    //          Or _if_ libdatachannel uses a thread pool internally, this could
+    //          be called once after rtcCleanup() or in some callback(?).
+    //   FIXME: android doesn't like this?
```

Comment:
let's drop this entire commit. I'll into a thread local key-based solution as suggested by android.

Reply:
ah you've reverted it later on.